designer/nodes/condition.mdx

```diff
@@ -0,0 +1,133 @@
+---
+title: "Condition"
+icon: "code-merge"
+iconType: "solid"
+---
+
+The Condition Node  allows you to create different paths in a conversation based on specific conditions. 
+
+This means you can make your assistant respond differently depending on the situation.
+
+<img 
+height="500"  
+src="/images/node-condition.png"
+/>
+
+Think of conditions like a cause and effect: "**If** this happens, **then** do that."
+
+
+<Tip>
+Example:
+- **If** I wake up early, **then** I will go to the gym. 
+- **If** I am hungry, **then** I will eat.
+</Tip>
+
+## Adding Conditional Paths
+The Condition Node can have multiple condition paths. These paths let your assistant handle various scenarios. 
+
+To create a new path, click on the **+ Add Condition** option.
+
+<img 
+height="500"  
+src="/images/condition-add.png"
+/>
+
+
+### Why Use Conditional Paths?
+Using conditional paths makes your assistant more dynamic and responsive. 
+Instead of giving the same response every time, your assistant can provide tailored replies based on the user's input or other variables.
+
+
+## Building a Condition
+Building a condition involves setting up rules that determine which path the conversation will take. 
+
+The Condition Node gives you the flexibility to check multiple criteria using two main elements:
+- Variable
+- Logic Group
+
+<img 
+height="500"  
+src="/images/condition-elements.png"
+/>
+
+
+### Variable
+A variable is a piece of information that your assistant knows or collects during the conversation. 
+You can use variables to check against specific values and determine the next step in the conversation.
+
+<img 
+height="500"  
+src="/images/condition-variable.png"
+/>
+
+Here’s how to work with variables:
+1. **Define a Variable:** Choose a variable from your variables list
```

Comment:
Define is probably not the correct term since the defining would have happened earlier in the conversation. Just select might work better.